### PR TITLE
Document guidelines for every field

### PR DESCRIPTION
**Pull request by axelboc**
With all the discussions that took place over the past few months, I thought it wise to add to the content guidelines for each field in _CONTRIBUTING.md_.

I wrote as much as I could, trying to remember all the decisions that we've made so far. I'm sure I've made a lot of mistakes and forgot some things. Please pick them apart 🔍 and don't hesitate to crush my feelings if you think something could be better written! ❤️ 

There's probably a bit of work to be done to apply these guidelines consistently in the deck (notably for capitals), and they'll probably have to be adjusted as we go. I'm still not a huge fan of having multiple capitals in the _Capital_ field, so I'm down with any guideline we can write to avoid that... 😂

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -31,45 +51,80 @@ Anki requires each note to have a unique identifier. When you add a note to the
...
+- alternative and former country names - e.g. _Also known as Timor-Leste_ (East Timor)
+- in rare cases only, general knowledge information - see Melanasia for an example
+
+The content of this field should be concise and consistent across notes. It may differ between languages, notably when dealing with alternative names.
+
+#### _Capital_ field
+
+Unless otherwise stated in [`TRANSLATION_SOURCES.md`](https://github.com/axelboc/anki-ultimate-geography/blob/master/TRANSLATION_SOURCES.md), we use the capital given in the [infobox](https://en.wikipedia.org/wiki/Template:Infobox_country#Examples) of the Wikipedia article for the country, in the language of the given deck.
+
+If a capital has alternative names, we take the **first listed name** in the Wikipedia article for the capital, in the language of the given deck. This corresponds typically to the title of the article and to the name used in the infobox of the country's Wikipedia article. Alternative names may be mentioned in the _Capital info_ field, when relevant.
```
Comment (aplaice) [inline CONTRIBUTING.md:84]: > If a capital has alternative names, we take the **first listed name** in the Wikipedia article for the capital

That's not what I used for [Laayoune](https://en.wikipedia.org/wiki/Laayoune) — there the first listed name is Laâyoune, but the article title and the name listed in the SADR infobox is just Laayoune. (I assume that you mean the first "bolded" name, not the name in the header?) Should I change it?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -31,45 +51,80 @@ Anki requires each note to have a unique identifier. When you add a note to the
...
+
+If multiple capitals are listed in a country's infobox, the following guidelines apply:
+
+- If the first capital is followed by a qualifier such as "official", "constitutional", "de jure", "claimed" or "political", it must be used alone in the _Capital_ field. The _Capital info_ field must then be used to detail the status and/or role of every capital - e.g. _While Dodoma is the official capital, Dar es Salaam is the de facto seat of government._
+- If government branches such as "executive" or "legislative" are the only qualifiers used, then the capitals must all be listed in the _Capital_ field, separated by comas - e.g. _Pretoria, Cape Town, Bloemfontein_ (South Africa). The _Capital info_ field must again be used to detail the role of every capital.
+- If no qualifiers are provided at all, then the capitals must all be listed in the _Capital_ field, separated by comas - e.g. _Santa Cruz de Tenerife, Las Palmas_ (Canary Islands). A concise explanation should then be provided in the _Capital info_ field.
```
Comment (aplaice) [inline CONTRIBUTING.md:90]: s/comas/commas/ (twice)

Comment (aplaice) [inline CONTRIBUTING.md:90]:
```suggestion
- If government branches such as "executive" or "legislative" are the only qualifiers used, then the capitals must all be listed in the _Capital_ field, separated by commas - e.g. _Pretoria, Cape Town, Bloemfontein_ (South Africa). The _Capital info_ field must again be used to detail the role of every capital.
- If no qualifiers are provided at all, then the capitals must all be listed in the _Capital_ field, separated by commas - e.g. _Santa Cruz de Tenerife, Las Palmas_ (Canary Islands). A concise explanation should then be provided in the _Capital info_ field.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -31,45 +51,80 @@ Anki requires each note to have a unique identifier. When you add a note to the
...
 
-### _Flag similarity_ field
+This field is used in notes that share:
 
-This field is used in the standard deck's _Flag - Country_ template.
+- the same exact capital - e.g. _London_ (United Kingdom and England);
+- capitals with very similar names - e.g. _Georgetown_ and _George Town_.
+
+The hints appear on _Capital - Country_ cards to avoid confusion or random guesses. They should
+convey as little information as possible to not giveaway the answers.
```
Comment (aplaice) [inline CONTRIBUTING.md:104]:
```suggestion
convey as little information as possible to not give away the answers.
```

I think that "give away" as a verb is two words, and [Wiktionary agrees](https://en.wiktionary.org/wiki/giveaway).

**Comment by aplaice** [comment]
That's really clear and thorough, while still being maximally concise!

Should the case of duplicate country names (#212) — currently only Saint-Martin/Sint Maarten in French and Spanish be mentioned or is it too much of a tiny corner-case?

(I can't really think of any other potential omissions.)

File: CONTRIBUTING.md (hunk 1 above)
Comment (axelboc) [reply CONTRIBUTING.md:84]: That's the rule that was already specified for country names, but I didn't think the page title could mismatch the first listed name... This seems like an oversight on Wikipedia's side, TBH. In the talk page, they discuss that `Laayoune` is more commonly used than `Laâyoune`, so it is indeed two valid and distinct spelling that should both appear in the introduction, IMO.

How about I just update the guideline to mention using the article title?

Comment (axelboc) [reply CONTRIBUTING.md]: Okay, I've [updated the guidelines](https://github.com/axelboc/anki-ultimate-geography/pull/255/commits/cfa011ed16c3844711c2933d5cbe2bc509779eca):

- For countries, we use the titles and we remove any disambiguation parenthetical, unless disambiguation is needed in the deck.
- For capitals, we use the titles and we remove any disambiguation parenthetical -- the hint field is there for that.

Comment (aplaice) [reply CONTRIBUTING.md]: I think that also works!

**Comment by aplaice** [comment]
Shouldn't this:

> In the very unlikely case that two countries share the same name in a given language,

be at the end of line 65, giving something like:

> If the title of the Wikipedia article contains a parenthetical portion for disambiguation purposes, it must be removed, unless the disambiguation is needed in the deck — e.g. _Saint-Martin (Antilles françaises)_ and _Saint-Martin (royaume des Pays-Bas)_ in the French deck — in the very unlikely case that two countries share the same name in a given language.

**Comment by axelboc** [comment]
Woops, thanks! I simplified the paragraph a little (I think).

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -31,45 +51,80 @@ Anki requires each note to have a unique identifier. When you add a note to the
...
 
-A note's _Flag similarity_ field contains a list of countries, each followed by a list of differences. For instance, the flag similarities for Iceland are written as follows: "Norway (red background, blue cross), Faroe Islands (white background, red and blue cross)". The list of differences must be **precise**, **clear** and **concise**. Advanced [vexillological](https://en.wikipedia.org/wiki/Vexillology) terms should be avoided.
+A note's _Flag similarity_ field contains a list of countries, each followed by a list of differences. For instance, the flag similarities for Iceland are written as follows: _Norway (red background, blue cross), Faroe Islands (white background, red and blue cross)_. The list of differences must be **precise**, **clear** and **concise**. Advanced [vexillological](https://en.wikipedia.org/wiki/Vexillology) terms should be avoided.
```
Comment (ohare93) [inline CONTRIBUTING.md:131]:
```suggestion
A note's _Flag similarity_ field contains a list of countries, each followed by a list of differences. For instance, the flag similarities for Iceland are written as follows: _Norway (red background, blue cross), Faroe Islands (white background, red and blue cross)_. The list of differences must be **precise**, **clear**, and **concise**. Advanced [vexillological](https://en.wikipedia.org/wiki/Vexillology) terms should be avoided.
```

Oxford comma to fully separate each item in the list.
image

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -89,25 +144,61 @@ Flag similarities are always **mutual**: if flag A is similar to flag B, then fl
...
+Two flags are then eligible for _Flag similarity_ information when they respect the two rules below:
 
-- Their differences all fit in the above classification. For instance, France and Italy are not elligible because their left bands are respectively blue and green, and "different colours in same position" does not appear in the classification.
+- Their differences all fit in the above classification. For instance, France and Italy are not eligible because their left bands are respectively blue and green, and "different colours in same position" does not appear in the classification.
 - Their classified differences form one the following combinations:
   - `1C 0M 0m {0+}n` = one critical difference and any number of negligible differences
   - `0C 1M {0-2}m {0+}n` = one major difference, up to two minor differences and any number of negligible differences
   - `0C 0M {0-3}m {0+}n` = up to three minor differences and any number of negligible differences
 
 Critical, major and minor differences should be listed in the _Flag similarity_ field. Negligible differences should be listed only when relevant, notably when two flags share nothing but negligible differences.
```
Comment (ohare93) [inline CONTRIBUTING.md:159]: > Critical, major and minor

Oxford comma

Comment (axelboc) [reply CONTRIBUTING.md:159]: Hmm this one hurts the flow of the sentence in my opinion. Oxford comma is good but doesn't have to be a hard rule ;)

**Pull request merged**